src/spurt/workflows/emcf/_tiling.py

```diff
@@ -0,0 +1,105 @@
+import json
+from pathlib import Path
+
+import numpy as np
+
+import spurt
+
+from ._settings import GeneralSettings, TilerSettings
+
+logger = spurt.utils.logger
+
+
+def get_tiles(
+    stack: spurt.io.SLCStackReader,
+    gen_settings: GeneralSettings,
+    tile_settings: TilerSettings,
+) -> None:
+    """Get the tiles for given stack."""
+    json_file = gen_settings.tiles_jsonname
+
+    # Generate tiles if file doesn't exist
+    if not json_file.is_file():
```

Comment:
This entirely boils down to whether this is production code that is deployed or a user playing with it on a laptop. Our most common use case is large scale processing - so dealing with multiple stacks on the same machine is a bit of an extreme use case. If that is the intention - our suggestion is the same as isce2 - do not process multiple interferograms/stacks in the same folder. Each stack gets its own directory structure. Having an explicit `resume` feels cosmetic to me.

Reply:
I think this should be more a workflow framework thing - with the framework having pre and post processing steps to determine whether a step needs to be done / not or if anything specific needs to be done in post processing. I think making this set of workflows to have super user friendly features might be out of scope. Incorporating a workflow framework and updating this workflow in future to use these type of features from the framework might be the way to go.

Reply:
I don't agree with all of your points here, but I think it's reasonable to consider this out-of-scope for the current effort. I'll open an Issue and we could continue to discuss whether/how this might be implemented later on.

>This entirely boils down to whether this is production code that is deployed or a user playing with it on a laptop. Our most common use case is large scale processing - so dealing with multiple stacks on the same machine is a bit of an extreme use case. 

Ideally I would support both use cases. Although the nominal production environment would have a single job per node, it doesn't seem extreme to me to want to run multiple successive jobs on a single machine, whether for testing or for some third party users' use case.

>If that is the intention - our suggestion is the same as isce2 - do not process multiple interferograms/stacks in the same folder. Each stack gets its own directory structure.

This is bad UX in my opinion. There's no reason to give the user this footgun. Having a `resume` option that's false by default makes the behavior explicit and makes the simplest/safest behavior the default while preserving the option to opt in to the more performant behavior.

Reply:
Agreed as out of scope